<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vhod</class>
 <widget class="QDialog" name="vhod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(5, 0, 176);


</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>          Войти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>        Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>       Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="access">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_in">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;P&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;N&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;M&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
